### ISSUE TITLE
Create transaction feature

### ISSUE THREAD

**Issue by levchmike** [paritytech/substrate-api-sidecar#1011]
Need new method (create transaction on server side)

**Comment by TarikGul** [comment]
We have been discussing potential solutions for this, and how we would want to implement such an endpoint. This will definitely be on our Roadmap in the future, but we won't see immediate adoption as there are some major feature's regarding contracts and xcm that we need to finish. 

Some notes on how something like this would be implemented:

- Required parameters or query params would be 1. `pallet`, 2. `method`, 3. arbitrary amount of `args`.

- The source of truth for constructing these transactions would probably be the `ApiPromise` registry. Txwrapper-core is an option but would probably not be efficient for transaction construction until metadata generation goes in.

**Comment by TarikGul** [comment]
After some discussion we have decided that this fits in as `feature-creep`. Constructing transaction payloads seems a bit out of scope for sidecar. There are libraries such as [txwrapper-core](https://github.com/paritytech/txwrapper-core) that can handle that logic. Sidecars main goal is a way to retrieve data from on chain and aggregate that data in a useful way (With the exception of `POST /transaction`). 

Closing.
